app/src/main/res/layout-sw600dp-port/onboarding_profile_type_fragment.xml

```diff
@@ -28,98 +28,86 @@
       app:customBackgroundColor="@{@color/component_color_onboarding_profile_type_background_color}"
       app:layout_constraintTop_toBottomOf="@id/profile_type_center_guide" />
 
+    <com.google.android.material.card.MaterialCardView
+    android:id="@+id/profile_type_learner_navigation_card"
+    style="@style/OnboardingProfileTypeNavigationCardStyle"
+    android:layout_width="wrap_content"
+    android:layout_height="0dp"
+    android:layout_marginStart="@dimen/tablet_shared_margin_medium"
+    android:layout_marginEnd="@dimen/tablet_shared_margin_medium"
+    app:layout_constraintBottom_toBottomOf="@id/profile_type_supervisor_navigation_card"
+    app:layout_constraintEnd_toStartOf="@id/profile_type_supervisor_navigation_card"
+    app:layout_constraintHorizontal_chainStyle="packed"
+    app:layout_constraintStart_toStartOf="parent">
```

Comment:
Is this indented correctly? Seems like it needs a reformatting.

Reply:
Reformatted the file